src/app/tests/suites/commands/interaction_model/InteractionModel.cpp

```diff
@@ -19,7 +19,20 @@
 #include "InteractionModel.h"
 
 using namespace chip;
-using namespace chip::app;
```

Comment:
Are there conflicts when using the whole app namespace? This seems a little cumbersome

Reply:
I do not remember what the conflicts were ... I think there are probably ways to make this better, there is only so much time available to spend here. Maybe when/if we touch this next we can update it to cleaner code.

Reply:
It is also unfortunate that this code as a stand-alone does not need these changes, just #34288 will need them. Once that code is integrated, we can start cleaning up things a bit more.